Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -214,7 +214,7 @@ end
 end
 
 macro checkSwitchToJITForLoop()
-    if WEBASSEMBLY_OMGJIT
+    if WEBASSEMBLY_BBQJIT or WEBASSEMBLY_OMGJIT
```

Comment:
Is this if needed anymore?

Reply:
For now it is, but soon armv7 will get omg too

Reply:
Maybe not. I will do that as a follow-up though, since  I don't want to accidentally break cloop